cypress/integration/admin.spec.js

```diff
@@ -1,61 +1,22 @@
-const { join: joinPath } = require('path')
-
-const ADMIN_LOGIN_URL = '/admin/login/'
-const ADMIN_URL = '/admin/'
-
-const CYPRESS_USER_JSON = joinPath(__dirname, '..', '..', 'cypress', '.cypress-user.json')
-
-Cypress.Commands.add('login', () => {
-  cy.visit('/admin/login/')
-  cy.get('[name=csrfmiddlewaretoken]')
-    .should('exist')
-    .should('have.attr', 'value')
-    .as('csrfToken')
-
-  cy.readFile(CYPRESS_USER_JSON)
-    .then(({username, password}) => {
-      cy.get('@csrfToken').then(function (token) {
-        cy.request({
-          method: 'POST',
-          url: ADMIN_LOGIN_URL,
-          form: true,
-          body: {
-            username,
-            password,
-            next: '/admin'
-          },
-          headers: {
-            'X-CSRFTOKEN': token,
-          },
-          followRedirect: false
-        }).then(response => {
-          expect(response.status).to.eql(302)
-          expect(response.headers).to.have.property('location')
-          expect(response.headers.location).to.not.contain('login')
-        })
-      })
-    })
-})
-
 context('Admin interface', () => {
   it('should redirect anonymous users to the login page', function() {
     cy.visit('/admin')
     cy.location().then(({pathname}) =>
-      expect(pathname).to.contain(ADMIN_LOGIN_URL))
+      expect(pathname).to.contain(Cypress.env('admin_login_url')))
   })
 
   it('should allow login', function() {
     // If this test fails, you are probably using `manage.py runserver` without
     // USE_TEST_DB=True, because the `cypress` user only gets created in the
     // test database.
     cy.visit('/admin')
-    cy.readFile(CYPRESS_USER_JSON)
+    cy.readCypressUserJSON()
```

Comment:
My personal preference, up to you: I avoid uppercase in variable names, so I’d call this `readCypressUserJson`. Makes it more readable when acronyms bump into each other in names, like `getHttpsJson`.